master/internal/sproto/task.go

```diff
@@ -98,6 +98,7 @@ type (
 	ValidateCommandResourcesRequest struct {
 		ResourcePool string
 		Slots        int
+		Command      bool
```

Comment:
1. `ValidateCommandResources` name does not make much sense any more if it's used for any task, not just commands. Maybe `ValidateResources...`?
2. This flag no longer means that the task is a command, but that it must be scheduled on a "single node" / single pod / single container. Maybe rename to `IsSingleNode`?

Reply:
Done!